lib/importer.js

```diff
@@ -159,10 +194,14 @@ module.exports = function(db, couchdb, concurrentDocLimit, changesLimit) {
       it is presumed you will do this yourself at a later stage.
       Returns a summary of edits, deletes, and the latest seq number from the batch
     */
-    importBatch: function() { return importChangesBatch(db, couchdb, concurrentDocLimit, changesLimit); },
+    importBatch: function() { return importChangesBatch(db, couchdb, concurrentDocLimit, changesLimit, source); },
     /*
       Persists a given seq value.
     */
-    storeSeq: function(seq) { return storeSeq(db, seq); }
+    storeSeq: function(seq) { return storeSeq(db, seq, source); },
+    /*
+      Returns a seq value for the given source.
+    */
+    getSeq: function(source) { return getSeq(db, source); }
```

Comment:
Is this just for tests? If so a convention we follow is prefixing the function with an underscore, as in this format that's the best tool for determining what is a real API and what is just for tests.